re_data_ui/public/index.html

```diff
@@ -1,43 +1,24 @@
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>re_data</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon"
+    href="data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAABg2lDQ1BJQ0MgcHJvZmlsZQAAKJF9kT1Iw0AcxV9TS0UqHewg4pChOlkUFXHUKhShQqgVWnUwufQLmhiSFBdHwbXg4Mdi1cHFWVcHV0EQ/ABxc3NSdJES/5cUWsR4cNyPd/ced+8AoVFlmtU1Bmi6bWZSSTGXXxHDrwhDQBSjCMnMMmYlKQ3f8XWPAF/vEjzL/9yfo1ctWAwIiMQzzDBt4nXiqU3b4LxPHGNlWSU+Jx4x6YLEj1xXPH7jXHJZ4JkxM5uZI44Ri6UOVjqYlU2NeJI4rmo65Qs5j1XOW5y1ao217slfGCnoy0tcpzmIFBawCAkiFNRQQRU2ErTqpFjI0H7Sxz/g+iVyKeSqgJFjHhvQILt+8D/43a1VnBj3kiJJIPTiOB9DQHgXaNYd5/vYcZonQPAZuNLb/o0GMP1Jer2txY+A6DZwcd3WlD3gcgfofzJkU3alIE2hWATez+ib8kDfLdCz6vXW2sfpA5ClrtI3wMEhMFyi7DWfd3d39vbvmVZ/PyqgcooGn7KpAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAA3XAAAN1wFCKJt4AAAAB3RJTUUH5QcCFQoWhqIISgAABHlJREFUWMO9l21MlWUYx3/3c57nnCMHStRAnRgiWrMUHQKaWgqNALGN1phWYm7VasuZK2NmL9rLMC2/1NhiNZeFH2zmKpHERCq0BBmSEhnvYo5OYIC8eM55nufug4icOMgBhf+3536u+/r/d1/XfV3XLfATjy75LMhw2VciRLxERgEzgDsBCXQA9QhZLiXHe7qteUWV6Z3++BVDGaTG5c4yDTIRrAEC/NTbJRD7TFN5P79sde2IBCxfvsc+rsv6joCNgMbI4AH5ofx3wrb8mhSX3wKSF+y/S6j6AZDLuA2Q8ItAe+xwaXrzkAJSor+ciyIOA9O4vbggpLEy73TGuUEFpMTsnwyeEiCM0cEFVRGx35564u/rC0r/mIPn4CiSA0zXTfPr5MjDtgECArq1t4FFjDrEA0pwW6ZXCFLjcmeZJpW3kO3DxRVdMyILTmY4FQDT5NUxJAcI0nTLGwAicd5eh2a3NEtJ4P+tbHaVFcnhAFSUNjNlWhDjJ9i52NBBdVUrUt6wvff+SUTMDiYgUKOxto3fypy4ruo3LVa6ywhVLTYl1Rc5gCNQY8OWWAB0j4mq9aUMf5xtYcdrxVhUhZdej2NudKjX3hZnNzm7yzhR2DSYAIdmszyiCMEKf86sP/l12AM0srIT+sjdboPuLg8Ak0IC2JK1jISUGTcrUPEqMN8fAc1/dfLu5p9oauhgSXwYpScu8fzmhYRMcWAYkpzdZRz5pgaP2+S++SG8sn0xIVMcPPdyNMWFTT7DIQRRClKE+yNg36fnqK9pQ9dNfixoRCiChxLvBuDAF79z6Ks/8bhNACrPOMneWQpAYJCVpQnTffo0JRFKb0sdEpdbur2+J091oKrXwlLcL86aVeHxtXPIfG9J39rMe4J9+lQEd6i9/Xz45UTcqOJab37EPTiNZzYuYGpYkJetYZiD51bvMDFuuAIaatpwuwysNgvJaZGcr2ylstzJpqePDLB1uwzfSShFuyKhbiQnoOsmeQeqAXg4NYIXM2MInmjHarOwPCmcTW8uwuMx6bzixu02BrsHtaoCZyQsHomI3JyzRC0MJWJ2MElpkSSlRXr935K1lG2biga/hlJWKKaUhSOtpz3dHjY/e5QfDtVhmt6pVF11mT0flQ8xD4pjYlX0JwGGEtgMBA1IEFUhPHI8AJearvQVGV8YP8HO7DkTsdosNNW301jXPpT+DoeqTRYAK2NysyW8MIbNCCH4OK/kyQ3Ktfto7AJcY8h/1SOVnX0DyXclGfVSyg/GjF6yo6B0TZPXROSUXdslomgMDv9nR4+W5XMoTYv9fKJLqqeAmaPE3iB1LTa/PP2fATMhwMGSda3CYlkF1I8CeZ0pzaT+5AMEAOT9urrKJvQYJMdvY8YX65qx+PvTa88PaEi+NhwsWdfq6NGSpGQr0H4L3D0C8Vaz0RlfcDLDOaLH6arofZN0ha0Cud7f1g20AXulInfln3rq4i29jvs/XBydaqIUSoJEzBPImf2qZ4dE1AlkhUQc63G4jhYVrb/qj9//AHTjoO+Kf38GAAAAAElFTkSuQmCC" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="theme-color" content="#000000" />
+  <meta name="description"
+    content="re_data is a web application that allows you to visualize your data in a more intuitive way." />
```

Comment:
Let's put line from github docs there: `re_data is an open-source data quality & observability framework for the modern data stack.`

Reply:
okau

Reply:
updated

https://github.com/re-data/re-data/pull/173/commits/650c503a6f8eaa9e629095a05d4df2735b4c775c